core/loadpoint.go

```diff
@@ -235,9 +235,9 @@ func NewLoadPointFromConfig(log *util.Logger, cp configProvider, other map[strin
 	lp.charger = cp.Charger(lp.ChargerRef)
 	lp.configureChargerType(lp.charger)
 
-	// ensure 1p setup for switchable charger (https://github.com/evcc-io/evcc/issues/1572)
-	if _, ok := lp.charger.(api.ChargePhases); ok && lp.Phases != 0 {
-		lp.log.WARN.Printf("ignoring phases config (%dp) for switchable charger", lp.Phases)
+	// TODO handle delayed scale-down
+	if _, ok := lp.charger.(api.ChargePhases); ok && lp.GetPhases() != 0 {
+		lp.log.WARN.Printf("ignoring phases config (%dp) for switchable charger", lp.GetPhases())
 		lp.setPhases(0)
 	}
```

Comment:
Überlege ob das nicht vielleicht so geschickter wäre:
```suggestion
	if _, ok := lp.charger.(api.ChargePhases); !ok && lp.GetPhases() == 0 {
		lp.log.WARN.Printf("using default phases config (3p) for non-switchable charger")
		lp.setPhases(3)
	}
```
Somit könnte man mit `phases: 0` bei 1p3p-fähigem Charger in den Automatikmodus schalten und mit `phases: 1` oder `phases: 3` den Modus manuell festlegen.

Reply:
Wie besprochen- andere Lösung erforderlich